app/components/Auth.js

```diff
@@ -22,7 +23,8 @@ const Auth = ({ children }: Props) => {
       const { user, team } = stores.auth;
       const cache = new CacheStore(user.id);
       authenticatedStores = {
-        settings: new SettingsStore(),
+        apiKeys: new ApiKeySettingsStore(),
+        memberSettings: new MemberSettingsStore(),
```

Comment:
Why not just make this `users: new UsersStore()` – I'm not sure that separating out 'settings' really has much value here.

Similar for ApiKeys.